cmplibrary/src/main/java/com/sourcepoint/cmplibrary/exception/ApiRequestSuffix.kt

```diff
@@ -0,0 +1,13 @@
+package com.sourcepoint.cmplibrary.exception
+
+internal enum class ApiRequestSuffix(
```

Comment:
Well actually 

> While suffix is a broad term used in linguistics and grammar to describe any morpheme added to the end of a word, postfix is a more specific term used in computer science and mathematics to describe a specific type of operator that is placed after the operand.

I'm being too picky, nevermind :)

Reply:
@Nevazhnovu good point

Reply:
In my understanding that should be a Suffix, not a Postfix. 

Postfix refers to a specific way of declaring operands, e.g.:
```
(1 2 +)
```

Whereas Suffix is anything that's added to the end of a word/sentence.